### DISCUSSION TITLE
i couldnt find some file by <leader><leader>

### DISCUSSION THREAD

**Discussion by shinonomeow** [LazyVim/LazyVim#1248] [Q&A]
my floder structure  is very simply

my_full_stack_project
  |- python_code
      |- 1.py
      |- 2.py

when i use <leader><leader> to open some file i never used ,i couldnt find them in result.
It displays 'git files' instead of 'find files',and there  are some file idk what is it like "100_\347\272\242"

I haven't made any changes to the configuration of the telescope.

Thank you for your help. I am still a beginne

**Comment by shinonomeow** [comment]
I understand where the problem lies now. It is due to the Chinese language

**Comment by shinonomeow** [comment]
How to solve the problem when the program name contains Chinese characters?i know it  is a bad naming convention.

**Reply by conanlm** [reply]
I've also encountered the same issue. Have you found a solution?
![wezterm-gui_sA50sJi6rk](https://github.com/LazyVim/LazyVim/assets/2777986/407c5526-e606-4bab-84e3-60cd21c0d375)
